mithril-stm/src/stm.rs

```diff
@@ -1633,5 +1634,15 @@ mod tests {
                 _ => unreachable!(),
             }
         }
+
+        #[test]
+        fn test_total_stake_core_verifier(nparties in 2_usize..30,
+                              m in 10_u64..20,
+                              k in 1_u64..5,) {
+            let params = StmParameters { m, k, phi_f: 0.2 };
+            let (_initializers, public_signers) = setup_equal_core_parties(params, nparties);
```

Comment:
Maybe you could fix the stake of the signers and assert that the result is exactly the value that you expect?